04_setup_ironic.sh

```diff
@@ -111,18 +111,18 @@ for IMAGE in ${IRONIC_IMAGE} ${VBMC_IMAGE} ${SUSHY_TOOLS_IMAGE} ; do
 done
 
 CACHED_MACHINE_OS_IMAGE="${IRONIC_DATA_DIR}/html/images/${MACHINE_OS_IMAGE_NAME}"
-if [ ! -f "${CACHED_MACHINE_OS_IMAGE}" ]; then
+if [ ! -f "${CACHED_MACHINE_OS_IMAGE}" -o ! -f "${CACHED_MACHINE_OS_IMAGE}.sha256sum" ]; then
   curl -g --insecure -L -o "${CACHED_MACHINE_OS_IMAGE}" "${MACHINE_OS_IMAGE_URL}"
   echo "${MACHINE_OS_IMAGE_SHA256} ${CACHED_MACHINE_OS_IMAGE}" | tee ${CACHED_MACHINE_OS_IMAGE}.sha256sum
```

Comment:
I think we probably want this to be unconditional too - we should use the latest checksum even if the checksum file does already exist.

Reply:
Yeah true, in theory the filename always changes when there's a new RHCOS build, but when you're locally hacking an image the checksum could change between runs with the same filename